stable/mission-control/values.yaml

```diff
@@ -184,9 +195,29 @@ missionControl:
 insightServer:
   replicaCount: 1
   name: insight-server
+  home: /opt/jfrog
```

Comment:
@amithins Why we need 2 mount points? Can we just use `mountPath: "/opt/jfrog/"`?

@eldada WDYT?

Reply:
@jainishshah17 
/opt/jfrog is passed as home for insight services, this will have default configuration, logs and any temp files the micro service will create. We need to persist only logs for insight services.

The mount point for each service is as follows :
- insight-services - logs are persisted
>insight-scheduler - /opt/jfrog/insight-scheduler/logs
insight-executor - /opt/jfrog/insight-executor/logs
insight-server - /opt/jfrog/insight-server/logs

- mission-control - data is persisted
>mission-control - /var/opt/jfrog/mission-control